drivers/serial/uart_hvc_xen.c

```diff
@@ -16,9 +16,10 @@
 #include <device.h>
 #include <init.h>
 #include <kernel.h>
-#include <logging/log.h>
 #include <sys/device_mmio.h>
 
+#define LOG_LEVEL CONFIG_UART_LOG_LEVEL
+#include <logging/log.h>
 LOG_MODULE_REGISTER(uart_hvc_xen);
```

Comment:
```suggestion
#include <logging/log.h>
LOG_MODULE_REGISTER(uart_hvc_xen, CONFIG_UART_LOG_LEVEL);
```

I strongly prefer using this variant of the `LOG_MODULE_REGISTER()` macro as it makes it much more clear, which log level is used by a given file.